src/nvme/tree.c

```diff
@@ -2102,16 +2102,17 @@ int nvme_init_ctrl(nvme_host_t h, nvme_ctrl_t c, int instance)
 		return ret;
 	}
 
-	ret = nvme_reconfigure_ctrl(h->r, c, path, name);
-	if (ret < 0)
-		return ret;
-
+	FREE_CTRL_ATTR(c->address);
```

Comment:
Isn't `nvme_init_ctrl` called only once and thus `c->address` should be NULL at this point anyway?